packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -171,26 +219,46 @@ class CollapsibleMenuView extends React.Component {
 
     if (this.hiddenStartIndex >= 0) {
       visibleChildren = React.Children.toArray(children);
-      hiddenChildren = this.collapsedMenuAlwaysShown
-        ? visibleChildren.splice(this.hiddenStartIndex).concat(prepopulatedBaseDivider).concat(hiddenChildren)
-        : visibleChildren.splice(this.hiddenStartIndex).concat(hiddenChildren);
+
+      if (this.props.isReversed) {
+        hiddenChildren = visibleChildren.splice(0, this.hiddenStartIndex + 2).concat(hiddenChildren);
+      } else {
+        hiddenChildren = this.collapsedMenuAlwaysShown
+          ? visibleChildren.splice(this.hiddenStartIndex).concat(prepopulatedBaseDivider).concat(hiddenChildren)
+          : visibleChildren.splice(this.hiddenStartIndex).concat(hiddenChildren);
+      }
+    }
+
+    const icon = useHorizontalIcon ? <IconEllipses /> : <IconEllipses className={cx('vertical-icon')} />;
```

Comment:
You would conditionally render the class name not the entire React component.  It should be easy with the classnames module.